### ISSUE TITLE
It's brokennn

### ISSUE THREAD

**Issue by whslater** [BoundingBoxSoftware/Materialize#18]
Download link on the site has stopped working, and seems to have been mixed with the unity release?

**Comment by schmildo** [comment]
Yes - downloading the windows 64 bit zip file from your website:
https://boundingboxsoftware.com/materialize/download/Materialize_1.78.zip

Gives this error:
Materialize - Unity 2017.4.3f1 (21ae32b5a9cb)
Materialze has encou terd an error and must clos.
A crash report named "Crash_2020-10-18-183816" has been generated. These files can be sent ot bou ding box software for anlaysix. woudl you like to oipen this foulder now? YES/NO
